packages/markup-frontstage/src/services/SavedView.ts

```diff
@@ -147,22 +220,97 @@ export const createSavedViewData = (vp: Viewport): SavedViewData => {
 
   const data: SavedViewData = {
     alwaysDrawn,
-    neverDrawn,
     categories,
+    changeSetId,
+    displayStyleProps: jsonStringify(viewState.displayStyle?.toJSON()),
```

Comment:
this will still break if viewState is undefined since its properties being accessed before the new function. Maybe just return at the beginning if viewState is undefined?

Reply:
viewState is checked at line 151 for undefined, I believe it should not break at the time displayStyle is being accessed, but I can add optional null chaining before it's property being accessed!

Reply:
done please!